src/app.js

```diff
@@ -13,7 +13,7 @@ import Footer from "./components/footer";
 import AccountDetails from "./views/account/details";
 import { OCAlertsProvider } from '@opuscapita/react-alerts';
 import history from './history';
-import JSBI from 'jsbi';
+import JSBI from 'jsbi/dist/jsbi-cjs';
```

Comment:
For some reason the browser gets the `umd` version, using even using a babel transformer doesn't get the job done. Some solutions mentioned tweaking the resolution alias on webpack to resolve this path on `jsbi` alias, which was not picked up from the config, so I figured, this does it fine for now.

Curious to hear what alternatives can you think of.

Reply:
Gotcha... It's very weird that we're having this issue with JABI—had never happened before to me. I think this might do for now yes, but I'll try and reproduce locally to see if there's a better fix. 👍

Reply:
That will be great!